src/App.js

```diff
@@ -18,30 +19,16 @@ function App() {
       <Route path='/register'>
         <WrappedRegistrationForm />
       </Route>
-      <Route path='/inventory'>
-        <Main />
-      </Route>
+      <PrivateRoute path='/inventory' component={Main} />
```

Comment:
Can we use the same format as the other routes for this?

`<PrivateRoute path='/inventory' component={Main} />`

becomes 

`<PrivateRoute path='/inventory'>
     <Main />
</PrivateRoute>`

Reply:
I tried that and it doesn’t work.

Reply:
Interesting. If that's the case I'm happy to close this issue.